docs/UserGuide.md

```diff
@@ -354,7 +358,12 @@ Format:
 * View price chart of properties: `stat property`
 * View price chart of buyers: `stat buyer`
 
+Visual Example:
+The below is the output from the command `stat`.
+
 ![stat example](images/StatUi.png)
+Since neither property nor buyer list is empty, you should see histograms of both properties and buyers.
+The price chart shows the number of properties and buyers for each price range.
```

Comment:
Unlike the other commands, this image does not actually follow from our samples, also it is quite hard to read the chart.
Maybe it is better to have a screenshot of just the chart. Not sure if this is a problem worth fixing